src/components/Icon/src/Icon.vue

```diff
@@ -15,12 +15,15 @@ import { MThemeKey, defaultTheme, resolveThemeableProps } from '@square/maker/co
 import RenderFn from '@square/maker/utils/RenderFn';
 
 const ICON_SIZES = {
-	medium: '16px',
-	large: '24px',
-	xlarge: '48px',
-	xxlarge: '64px',
+	small: '16px',
+	medium: '24px',
+	large: '32px',
+	xlarge: '40px',
+	xxlarge: '48px',
 };
 
+const widthValidator = cssValidator('width');
```

Comment:
Guessing there's no difference between `cssValidator('width');` and `cssValidator('height');` so this is probably fine?

Reply:
i thought of doing this at first but after checking MDN it seems like `width` and `height` properties accept the same set of values. i'll add a code comment so it's more clear.